acs-category-theory-notes.tex

```diff
@@ -416,23 +480,252 @@ \subsection{The dual problem}
+There is also special terminology for orders which obeys these axioms:
+
+\begin{definition} Partial orders and preorders
+\begin{itemize}[noitemsep,topsep=0pt]
+\item $(P, \leq)$ is a \emph{preorder}, or \emph{quasiorder} if $\leq$ is
+reflexive and transitive
+\item $(P, \leq)$ is a \emph{partial order} if is a preorder (reflexive and
+transitive) and $\leq$ is also antisymmetric
+\end{itemize}
+\end{definition}
+
+We may define an algebraic structure on top of a partially ordered set:
+
+\begin{definition}
+A partial order $(P, \leq)$ augmented with an idempotent, commutative and
+associative binary operation $\lor$\footnote{\textbf{join} or
+    \textbf{disjunction}} that satisfies $x \leq y \iff x \vee y = x$ is a
+\textbf{join semilattice}.
+
+It may also have a least (bottom) element $\bot \in L$ satisfying
+$\forall x \in L,~\bot\lor x = \bot$.
+\end{definition}
+
+\subsection{Exercises}
+
+We can now diagram what we desire from an initial solution to a structure of
+this kind. if $S$ is a set, then we want to construct an initial join
+semilatice, as follows:
+% Create an example environment?
+\begin{align*}
+    \textbf{data:}\qquad& S \overset{f_1}{\longrightarrow}L \qquad\qquad
+    \text{$(L,\lor)$ a join semilatice}\\
+    \textbf{criteria:}\qquad& \begin{tikzcd}[ampersand replacement=\&]
+        S \arrow[r, "f_1"] \arrow[dr, swap, "f_2"] \& 
+        \tilde{S}
+        \arrow[d, "\exists ! h"] 
+        \& (\tilde{S},\lor_{\tilde{S}},) 
+        \arrow[d, "\text{$h$ a homomorphism}"] 
+        \& \text{a universal (initial) join semilatice}
+        \\
+        {}  
+        \& L \& (L,\lor_L) \& \text{any join semilatice}
+    \end{tikzcd}\\
+    &\text{where $h$ must preserve structure, \ie~$h(x\lor_{\tilde{S}} y) = h(x)
+    \lor_L h(y)$}
+\end{align*}
+The exercise is to find the initial solution here, but we will start with some
+intuition. The join semilatice generated from $S$ must contain each element
+$x\in S$. Similarly, for any two, or three elements in $S$, it must contain the
+disjunction of all of them. Since the operator is commutative, the order doesn't
+matter, and since it's idempotent, only one copy of each element of $S$ can
+exist. The picture looks something like this:
+\begin{align*}
+x  &\qquad\leftrightarrow\qquad \setof{x} \subseteq S \\
+x \lor y &\qquad\leftrightarrow\qquad \setof{x,y} \subseteq S \\
+x \lor y \lor z &\qquad\leftrightarrow\qquad \setof {x,y,z} \subseteq S \\
+ &\qquad\vdots \\
+x_1 \lor \cdots \lor x_n &\qquad\leftrightarrow\qquad \setof{x_1, \cdots x_n}
+\subseteq S
+\end{align*}
+
+Thus, we might guess that the universal initial solution $\tilde{S}$ is the
+collection of finite subsets of $S$:
+$\mathcal{P}_{\mathrm{fin}}(S) = \setof{X \suchthat
+X \subseteq_\mathrm{fin} S}$, with the join operator $X \lor Y = X \cup Y$.
+This is close, but not entirely correct.
+
+\begin{exercise}
+There is a `bug' in this solution. Find the actual initial solution.
+\end{exercise}
+
 \end{document}
```

Comment:
Thanks for these updates. My request is that future updates are in their own branch, not mixed in with new lecture content.